docs-user/guide-filtering-call-trees.md

```diff
@@ -74,6 +74,16 @@ Merging takes a call node and removes it from the call tree. Any self time for t
 
 Focusing on a function or call node removes all of the ancestor call nodes—the children call nodes remain. If a stack does not contain that function or node, then it is removed. This effectively focuses on a subtree or a set of subtrees on the call tree.
 
+### Focus on Category
+
+Focusing on the functions that belong to the same category as the selected function thereby removing all other functions.
```

Comment:
```suggestion
Focusing on the nodes that belong to the same category as the selected node thereby merging all nodes that belong to another category.
```

I'm not sure `Focus` is the right terminology here. We use `focus` for other transforms, where it drops everything else, but in this case we're not dropping, rather we're merging. Is that really what we want? Maybe we want a focus operation really for this one (that is we'll drop all other functions)?

I think `merging` (or rather `collapsing`) makes sense for the other transform though.

Reply:
I thought of it more and looked at other focus operations, and now I think `Focus` is the right word;

The suggested change above is still to be done though :-) We're merging _nodes_, not _functions_.